renderdoc/core/replay_proxy.cpp

```diff
@@ -1707,6 +1707,21 @@ bool ReplayProxy::SendReplayCommand(ReplayProxyPacket type)
   return true;
 }
 
+// Modify the params that are used when getting the texture data for replay on the current driver.
+void ReplayProxy::ConvertProxyTextureIfNeeded(FetchTexture &tex, GetTextureDataParams &params)
+{
+  if(m_Proxy->IsTextureFormatSupported(tex.format))
+    return;
+
+  tex.format.special = false;
+  tex.format.compCount = 4;
+  tex.format.compByteWidth = 1;
+  tex.format.compType = eCompType_UNorm;
+
+  params.forceRGBA8unorm = true;
+  params.whitePoint = 1.0f;
```

Comment:
In the common code in replay_proxy.cpp im now switching on the remap type to determine the component counts and widths etc. This can be shared across APIs.

The D3D-specific code in dxgi_common.cpp is where the remap type is generated.

All depth/stencil formats now remapping with REMAP_D32S8, which is unimplemented.

For the correct whitepoint I dont think we have LDR/HDR enums yet so sticking with 1.0.